src/xunit.performance.analysis/Program.cs

```diff
@@ -311,22 +353,28 @@ private static string EscapeCsvString(string str)
 
         private static void WriteTestResultsCSV(Dictionary<string, Dictionary<string, TestResult>> testResults, string csvOutputPath)
         {
+        #if LINUX_BUILD
```

Comment:
it seems the Linux approach here (using FileStream) could work for Windows too, so we get rid of one #if